tsan_env_suppressions

```diff
@@ -0,0 +1,8 @@
+# For ThreadSanitizerSuppressions
+# https://github.com/google/sanitizers/wiki/ThreadSanitizerSuppressions
+
+race:*/ring_buffer.h
+
+deadlock:*/store.h
```

Comment:
The way to specify ideally should be like `*/src/kv/store.h` , but it doesn't work depending on (maybe) working directory.
Sometimes TSAN somehow thinks  `<CCF>/src/kv/store.h` as `<CCF>/src/kv/../kv/store.h` for examples. In that case the regex doesn't match.